zephyr/src/supp_api.c

```diff
@@ -65,27 +69,155 @@ static int send_wpa_supplicant_dummy_event(void)
 	return send_wpa_supplicant_event(&msg);
 }
 
+/* Lifted from zephyr shell_utils.c to handle escapes */
+static inline uint16_t supp_strlen(const char *str)
+{
+	return str == NULL ? 0U : (uint16_t)strlen(str);
+}
 
-static inline struct wpa_supplicant * get_wpa_s_handle(const struct device *dev)
+static char make_argv(char **ppcmd, uint8_t c)
 {
-	struct wpa_supplicant *wpa_s = NULL;
-	int ret = k_sem_take(&wpa_supplicant_ready_sem, K_SECONDS(2));
+	char *cmd = *ppcmd;
+	char quote = 0;
```

Comment:
AFAICT, quote will never have any value other than 0 in this function, is that intended?

Reply:
Yes, this can be removed.

Reply:
As its a copy, I left the code unchanged.